packages/app/lib/services/imageUploadService.ts

```diff
@@ -0,0 +1,29 @@
+import { apiUrl } from '../utils/utils';
+
+export const imageUpload = async ({
+  data,
+  authToken,
+}: {
+  data: FormData;
+  authToken: string;
+}): Promise<string> => {
+  try {
+    const response = await fetch(`${apiUrl()}/upload`, {
+      method: 'POST',
+      cache: 'no-cache',
+      headers: {
+        Authorization: `Bearer ${authToken}`,
+      },
+      body: data,
+    });
+
+    if (!response.ok) {
+      throw 'Error uploading image';
+    }
+
+    return (await response.json()).data;
+  } catch (e) {
+    console.log('error in upload image', e);
```

Comment:
Bit picky on this, but it is quite a bad practice to catch and then directly throw again. I think it it best to remove the try / catch, and let `imageUploadAction` catch the error.